packages/ui/components/Pages/Workspace/Multipleprogram.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { Button } from 'components/Button';
+import { Card } from 'components/Card';
+import { Icon } from 'components/Icon';
+import { LetterPic } from 'components/LetterPic';
+import styles from 'styles/multipleprogram.module.css';
+import programs from 'fixtures/programs.json';
+import selectedprograms from 'fixtures/selectedprograms.json';
+
+function MultipleProgram() {
+  return (
+    <>
+      <div className="flex px-4">
+        {programs.map((p) => {
+          return (
+            <Card
+              key={p.title}
+              className={p.className}
+              title={p.title}
+              description={p.description}
+              icon={
+                <div className="flex -space-x-1 overflow-hidden">
+                  <LetterPic
+                    letter="F"
+                    size="sm"
+                    color="primary"
+                    className="text-sm text-gray-600 inline-block h-6 w-6 rounded-full ring-2 ring-white"
+                  />
+                  <LetterPic
+                    letter={
+                      <img
+                        className="inline-block h-6 w-6 rounded-full ring-transparent"
+                        src="https://images.unsplash.com/photo-1500648767791-00dcc994a43e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2.25&w=256&h=256&q=80"
+                        alt=""
```

Comment:
@pipesort123TP need to extract out as an hardcoded array, and should be replaced later by backend data